### PR TITLE
[DEVOPS-6] Deploy explorer to testnet

### PR DESCRIPTION
**Pull request by domenkozar**
- merge explorer deployment into cardano (it has to use the same commit anyway)
- modify package generation to comply with `cardano-sl-0.4` stack.yaml
- bump nixpkgs version in hydra to match the production

### CODE REVIEW

**Comment by deepfire** [comment]
@domenkozar, do I get it right that you have successfully managed to transition to a `cabal2nix`-generated `cardano-sl` derivation, but for `cardano-sl-explorer` we have no such luxury?

What is the difference between them?

**Comment by domenkozar** [comment]
We build both using cabal2nix. Some dependencies (socket-io) are special since `.cabal` is not present in top-level folder.

**Comment by deepfire** [comment]
But you have removed `pkgs/cardano-sl.nix`, and not `pkgs/cardano-sl-explorer.nix`?

**Comment by domenkozar** [comment]
It's now a JSON: https://github.com/input-output-hk/iohk-nixops/pull/29/files#diff-461f3277aa17b1a12e5ea5bcb24d48c5

Then I fetch the source with `fetchgit`: https://github.com/input-output-hk/iohk-nixops/pull/29/files#diff-5712e736e0de6ba170577f8472c398e9R13

And then reuse that `src` in all further packages within that repo. Same for `socket-io`.

File: pkgs/generate.sh [hunk 1]
```diff
@@ -4,22 +4,25 @@
...
-
 cabal2nix https://github.com/thoughtpolice/hs-ed25519.git --revision da4247b5b3420120e20451e6a252e2a2ca15b43c > ed25519.nix
 cabal2nix https://github.com/avieth/network-transport.git --no-check --revision f2321a103f53f51d36c99383132e3ffa3ef1c401 > network-transport.nix
 cabal2nix https://github.com/avieth/network-transport-tcp.git --no-check --revision ca42a954a15792f5ea8dc203e56fac8175b99c33 > network-transport-tcp.nix
 cabal2nix https://github.com/ndmitchell/derive.git --no-check --revision 9b564c23543d92757168581beb832f4dc0db223b > derive.nix
 cabal2nix https://github.com/input-output-hk/cardano-crypto --no-check --revision 838b064d8a59286142aa2fe14434fe7601896ddb > cardano-crypto.nix
 cabal2nix https://github.com/haskell-crypto/cryptonite.git --no-check --revision 6440a7ebab7d85612e47099017bee0da6339af05 > cryptonite.nix
-cabal2nix https://github.com/input-output-hk/cardano-sl-explorer.git --no-check --revision 3ff32b51fef5a89722598a65acc47218edd44683 > cardano-sl-explorer.nix
+cabal2nix https://github.com/input-output-hk/cardano-sl-explorer.git --no-check --revision 174888ef5647b60a40131fa77edc765c566d4aed > cardano-sl-explorer.nix
```
Comment (jmitchell) [inline pkgs/generate.sh]: The latest revision deployed, as of 18:00 UTC on 26 April, is `a7f23de9bb8253e61a69d9acc3a1d5ad77680973`.

Comment (domenkozar) [inline pkgs/generate.sh]: Done, thanks

**Pull request merged**